localization/english/event_localization/wc_demon_hunter_events_l_english.yml

```
@@ -0,0 +1,62 @@
+﻿l_english:
+ 
+ demon_hunter_0001_title:1 "A Destiny of Flame"
+ demon_hunter_0001_desc:0 "This may be a drastic decision I have taken, but I see no other choice.\n\nI take a deep breath while walking to our prisons at night. I could already smell the foul and evil nature of the [GetTrait('being_demon').GetName(GetNullCharacter)] which can only make one frown.\n\nAnd there lies a violent, mana-ridden and pathetic [first_demon_prey.GetFirstName].\n#XB "Ha... Here to finish me off? Ignorant fool! Our kind never passes away like you weaklings... and I will be back to have vengeance! Your loved ones and your nation, all will face the wrath of the Burning Legion!"#!, [first_demon_prey.GetFirstName] says.\n\nIt comforts me knowing that what I am about to do is nothing short of huge service to my kind. I hold [first_demon_prey.GetFirstName] by its face in one hand as it struggles in its chains and with my blade I strike at the neck to sever [first_demon_prey.GetFirstName]'s head and the struggling stops.\n\nAs I drink the blood that drips from the lifeless head, I already feel the powers of the [GetTrait('being_demon').GetName(GetNullCharacter)] flowing through me."
+ demon_hunter_0001_A:0 "...and then, I carve into its heart"
+
+ demon_hunter_0002_title:1 "A Destiny of Flame"
+ demon_hunter_0002_desc:0 "As the still beating heart of the [GetTrait('being_demon').GetName(GetNullCharacter)] still thumps in my hands, I had learnt that they pass away from whence they came... only to return to our worlds at will...\n\nBut I #italic know #!that they cannot come back if they never return to their world.\nI grab the heart and consume it with its filthy smell and energy and all, binding [first_demon_prey.GetFirstName] to my soul. By the time I finish consuming it, I not only feel the dark energies inside me but also my surroundings suddenly change...\n\nNo... is this a vision of the past?... my eyes burn trying to grasp what I see... I see numerous [GetTrait('being_demon').GetName(GetNullCharacter)]s marching onto countless worlds like ours and destroying them through sheer force and sometimes even from within...\n\nSo much bloodshed, kind worlds slain and only flames that remain... the pain resonates within my soul... I wish for them to stop but they keep going and going and going...\nIn my most desperate urge to stop them, my arms move on their own to stop the pain and visions, I rip my own eyes out."
```

this text needs improvement. here is my suggestion

```suggestion
  demon_hunter_0002_desc:0 "As the still-beating heart of the [GetTrait('being_demon').GetName(GetNullCharacter)] thumps in my hands, I realize how to stop the monster's cycle of rebirth in the [twisting_nether|E]:\nI consume the foul thing, energy and all, binding [first_demon_prey.GetFirstName] to my soul. As soon as I finish consuming the heart, I feel its dark energies fill my being.\n\n Suddenly, my surroundings change and my eyes burn as I try to grasp what I see: Numerous [GetTrait('being_demon').GetName(GetNullCharacter)]s marching onto countless worlds like ours, destroying them either through brute force or subtly from within.\n\nPlacid worlds reduced to ash and flame... I feel their pain resonate within my soul... I wish for the visions to stop but they keep going and going and going...\nAbandoning myself to my most desperate urge to stop them, I rip my own eyes clean from their sockets..."
```

---

I'll revisit this, thanks.